src/lib/viewers/image/ImageViewer.js

```diff
@@ -513,6 +514,15 @@ class ImageViewer extends ImageBaseViewer {
      */
     handleAnnotationControlsClick({ mode }) {
         const nextMode = this.annotationControlsFSM.transition(AnnotationInput.CLICK, mode);
+
+        if (nextMode === AnnotationMode.REGION) {
+            if (!this.cache.get(IMAGE_FTUX_CURSOR_DISABLED_KEY)) {
+                this.cache.set(IMAGE_FTUX_CURSOR_DISABLED_KEY, true, true /* useLocalStorage */);
```

Comment:
What is the reasoning for this block comment? Does it affect webpack in some way?

Reply:
I don't think it affects webpack, I initially kept it there because I thought it was used to indicate to the user what the parameter was for. I'm open to removing it as I'm seeing instances of where it's used as well as not used.